pom.xml

```diff
@@ -162,6 +162,26 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
```

Comment:
Is this addition to define Checkstyle as a dependency? If so, shouldn't it be made so with the `dev` scope?

Reply:
This makes CheckStyle a plugin for development (different from a dependency)

Reply:
Gotcha. We appear to have a bunch of plugins already, this is probably fine.